test/format/backup.c

```diff
@@ -657,6 +709,12 @@ backup(void *arg)
     if (incremental != 0)
         check_copy();
 
+    /*
+     * Copy format-specific files into the BACKUP.copy directory so that test/format can be run on
+     * the BACKUP.copy database for verification. These include CONFIG and any CONFIG.keylen* files.
+     */
+    copy_format_files(session);
```

Comment:
I think we only need to do this once, the first time. I don't *think* any of the CONFIG* files change once created. Therefore, this call should be in the same conditional that initially creates the directories. Look for where we call `testutil_create_backup_directory`. I also think it does not hurt to copy them into both directories.

Reply:
Thanks for the suggestion Sue